src/sys/windows/selector.rs

```diff
@@ -42,13 +42,9 @@ struct SelectorInner {
     /// Primitives will take buffers from this pool to perform I/O operations,
     /// and once complete they'll be put back in.
     buffers: Mutex<BufferPool>,
-}
 
-pub struct Registration {
-    selector: Option<Arc<SelectorInner>>,
-    token: Token,
-    opts: PollOpt,
-    interest: EventSet,
+    /// A list of registered level triggered `IoEvent`s
+    level_triggered: Mutex<LinkedList<IoEvent>>,
```

Comment:
It might be interesting to move this mutex inside `LinkedList` eventually as in theory `list.get(entry)` shouldn't require much synchronization (or there could be atomic ops in play).

That being said I would want to think through that much more thoroughly and this seems totally fine for now (especially as `level` is less common than `edge`)